From 6ee0100122d6c248d7390caea8d7dc7a0d7eef70 Mon Sep 17 00:00:00 2001
From: Tomas Henzl <thenzl@redhat.com>
Date: Thu, 16 Oct 2014 16:02:11 -0400
Subject: [scsi] mpt3sas: Bump mpt3sas driver version to 04.100.00.00

Message-id: <1413475335-14737-29-git-send-email-thenzl@redhat.com>
Patchwork-id: 97605
O-Subject: [RHEL7.1 e-stor PATCH 28/32] mpt3sas: Bump mpt3sas driver version to 04.100.00.00
Bugzilla: 1088524
RH-Acked-by: Maurizio Lombardi <mlombard@redhat.com>
RH-Acked-by: Rob Evers <revers@redhat.com>

Bump mpt3sas driver version to 04.100.00.00.

Signed-off-by: Sreekanth Reddy <Sreekanth.Reddy@avagotech.com>
Reviewed-by: Martin K. Petersen <martin.petersen@oracle.com>
Signed-off-by: Christoph Hellwig <hch@lst.de>
(cherry picked from commit 70d8c86ebeaca552801500d20d5f2c6cc235cabd)
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/scsi/mpt3sas/mpt3sas_base.h b/drivers/scsi/mpt3sas/mpt3sas_base.h
index ca4a7cc..40926aa 100644
--- a/drivers/scsi/mpt3sas/mpt3sas_base.h
+++ b/drivers/scsi/mpt3sas/mpt3sas_base.h
@@ -70,8 +70,8 @@
 #define MPT3SAS_DRIVER_NAME  "mpt3sas"
 #define MPT3SAS_AUTHOR "LSI Corporation <DL-MPTFusionLinux@lsi.com>"
 #define MPT3SAS_DESCRIPTION "LSI MPT Fusion SAS 3.0 Device Driver"
-#define MPT3SAS_DRIVER_VERSION  "03.100.00.00"
-#define MPT3SAS_MAJOR_VERSION  3
+#define MPT3SAS_DRIVER_VERSION  "04.100.00.00"
+#define MPT3SAS_MAJOR_VERSION  4
 #define MPT3SAS_MINOR_VERSION  100
 #define MPT3SAS_BUILD_VERSION  0
 #define MPT3SAS_RELEASE_VERSION 00
-- 
1.7.1